R/pkg/R/DataFrame.R

```diff
@@ -1187,6 +1187,69 @@ setMethod("dapply",
             dataFrame(sdf)
           })
 
+#' gapply
+#'
+#' Apply a function to each group of a DataFrame. The group is defined by an input
+#' grouping column.
```

Comment:
only a single column? 

If any limitation exists, you also need to document it here.